config/ci-rocm.yaml

```
@@ -0,0 +1,296 @@
+defaults: &defaults
+  dirs:
+    code: code/{name}
+    venv: venv/{name}
+    data: data
+    runs: runs
+
+  venv:
+    type: virtualenv
+
+  voir:
+    --forward:
+      - "#stdout"
+      - "#stderr"
+      - "#end"
+      - loss
+      - compute_rate
+      - train_rate
+      - loading_rate
+      - gpudata
+      - gpu_time
+      - cpu_dispatch_time
+      - cpu_walltime
+      - tflops
+    # --loading-rate: true
+    --poll-gpu: 10
+
+
+torchvision: &torchvision
+  <<<: *defaults
+
+  group: torchvision
+  install_group: torchvision
+
+  dirs:
+    code: code/torchvision
+    venv: venv/torchvision
+
+  plan:
+    method: per_gpu
+
+  pip:
+    args:
+      - --extra-index-url
+      - https://download.pytorch.org/whl/rocm5.2/
+
+  voir:
+    --stop: 30
+    --metric: step
+    --metric-interval: 1s
+    --metric-skip: 5s
+
+  argv:
+    --with-amp: true
+    --lr: 0.01
+
+  definition: ../benchmarks/torchvision
+
+
+torchbench: &torchbench
+  <<<: *defaults
+
+  group: torchbench
+
+  dirs:
+    code: code/torchbench
+    venv: venv/torchbench
+
+  plan:
+    method: per_gpu
+
+  definition: ../benchmarks/torchbench
+
+  pip:
+    args:
+      - --extra-index-url
+      - https://download.pytorch.org/whl/rocm5.2/
+    constraints:
+      - gym < 0.26.0
+
+  voir:
+    --metric: step
+    --metric-interval: 1
+    --metric-skip: 9
```

Changes in torchbenchmark require this adjustment.

```suggestion
    --stop: 20
    --metric: step
    --metric-interval: 1s
    --metric-skip: 5s
```